src/Spec-Inspector/EyeInspector.class.st

```diff
@@ -11,6 +11,9 @@ Class {
 		'list',
 		'description'
 	],
+	#classInstVars : [
+		'useAutoRefresh'
```

Comment:
This variable should not be here.

Reply:
uh? 
I didn't add them.

Reply:
I removed them in a previous commit. Maybe Iceberg got mixup?